yamls/clusters/docker-cluster-lifecycle-hooks.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: "docker-cluster-lifecycle-hooks"
```

Comment:
Would it be better if we namespaced this and added a selector in the extension config? I think that will give this section a smaller blast radius e.g. if someone adds blocking but then moves on without cleaning up.

Reply:
Good idea. The only reason I did not do it is we will need another clusterclass in the new namespace for this to work. We could simple include the clusterclass definition also in  this file.  
That should be okay, right?

Reply:
Ah good point. Yup should be okay to keep it simple

Reply:
Seems fixed